src/frontend/screens/Settings/components/WinePrefixesBasePath.tsx

```diff
@@ -19,20 +21,59 @@ const WinePrefixesBasePath = () => {
     'defaultWinePrefix',
     ''
   )
+  const [lastValidPrefix, setLastValidPrefix] = useState(defaultWinePrefix)
+
+  function handlePathChange(val: string) {
+    setDefaultWinePrefix(val)
+    if (val) {
+      setLastValidPrefix(val)
+    }
+  }
```

Comment:
Instead of storing `lastValidPrefix` (which the name can be a bit misleading, it doesn't mean this is valid) I imagine this logic for `handlePathChange`:

```ts
function handlePathChange(newPath: string) {
  if (newPath) {
    // not empty, all good
    setDefaultWinePrefix(newPath)
  }  else {
    // empty, show an alert/warning saying it will be reverted to the previous value
    // reset the value to the original config value
  }
}
```

But I actually think we have a problem with the PathSelectionBox component because we have no simple way of resetting the value (we would have to re-render the component cause it uses the internal tmpPath state).

I imagine the `onPathChange` function that we pass with the props could be updated to return a string (instead of void) that the component should set the field to, and instead of simply calling that function in the `onBlur` callback, we could wrap it to call it and use the return value to update with `setTmpPath(theReturnValueOfOnPathChange)`.